Jenkinsfile

```diff
@@ -54,9 +54,9 @@ def clone (name) {
 
 def test (name) {
     if (name == 'dmd')
-        sh "make -j 4 -f posix.mak test MODEL=64"
+        sh "make -f posix.mak test MODEL=64 --jobs=4"
```

Comment:
[Dedibox® LT 2016 - Online.net](https://www.online.net/en/dedicated-server/dedibox-lt)
1x Intel® Xeon® E3 1245 v5 
4 C / 8T @3.5 Ghz cache L3 8MB, x64, VT

Parallelizing those make sense to reduce the latency of test results. I don't think we have that many PRs to test at the same time. And I also guess that it's more efficient to run druntime->phobos->dmd tests one after another rather than in parallel, can check that.